GliaWidgets/Sources/ViewModel/Chat/Data/ChatMessage.swift

```diff
@@ -25,7 +25,8 @@ enum ChatMessageSender: Int, Codable {
 }
 
 class ChatMessage: Codable {
-    let id: String
+    typealias MessageId = String
```

Comment:
what is the purpose of this typealias? Can we remove it?

Reply:
This typealias is like a hint for devs that `historyMessageIds` set should store only message ids, because compiler still allows you to store any string there.

Reply:
@rasmustautsglia Egor is making the exact point. It is a hint that it's not just a regular string, but message id. Why do you think it has to be removed?